tests/python_on_whales/components/test_image.py

```diff
@@ -24,18 +30,20 @@ def test_image_repr():
     docker.image.remove(["busybox:1", "busybox:1.32"])
 
 
-def test_image_remove():
-    docker.image.pull("busybox:1", quiet=True)
-    docker.image.pull("busybox:1.32", quiet=True)
-    docker.image.remove(["busybox:1", "busybox:1.32"])
+@pytest.mark.parametrize("ctr_client", [docker_client, podman_client], indirect=True)
+def test_image_remove(ctr_client: DockerClient):
```

Comment:
This is a bit counter-intuitive too, we're specifying the parameterised values for `ctr_client` as parameters that have string values ("DOCKER_TEST_FLAG" and "PODMAN_TEST_FLAG") but then what we actually get is `DockerClient` instances. I don't think this is how paramaterisation normally works (even when using `indirect=True`)?

Reply:
This is the effect of `indirect=True` it means the argument is given to a fixture and then the fixture yields the value given to the test.